cea/demand/demand_main.py

```diff
@@ -133,7 +133,7 @@ def write_totals_csv(building_properties, locator, gv):
 
 def thermal_loads_all_buildings(building_properties, date, gv, locator, num_buildings, usage_schedules,
                                 weather_data):
-    for i, building in enumerate(building_properties.list_building_names()):
+    for i, building in enumerate(building_properties.list_building_names()[:1]):
```

Comment:
here, again: this introduces a but into the system. I wonder, while testing, maybe just create a function that does a single `thermal_loads.calc_thermal_loads` call would work better? You could integrate this into the unit-testing suite...